docker-bake.hcl

```
@@ -40,10 +39,17 @@ variable "TARGETS" {
 function "tags" {
   params = [image]
   result = [
-    "${REGISTRY}${ORGANIZATION}/${image}:newly-build",
+    "${REGISTRY}${ORGANIZATION}/${image}${VERSION}",
```

```suggestion
    "${REGISTRY}${ORGANIZATION}/${image}:${VERSION}",
```

Should this be with a `:`?  I am also not sure we still need the "<image>-meta" target after the refactoring. I think maybe the `tags` function can directly put into the `<image>` target?

---

And a nitpick.

`${REGISTRY}${ORGANIZATION}` -> `${REGISTRY}/${ORGANIZATION}`

with using `ghcr.io` instead of `ghcr.io/` in `env.hcl` probably more clear?

---

Good catch. I actually tried both of those things and ran into issues. The way this currently works in CI is that the $image already includes both the registry and VERSION so I cannot include the separators here. Maybe something to improve in the future, but I didn't find an obvious way to do it.

---

I have a feeling that the `VERSION` is not actually used, right?

---

I think it is still use when you run the build locally.